src/content/docs/contributing.mdx

```diff
@@ -0,0 +1,5 @@
+---
+title: Contributing
+---
+
+Until we get our contribution guides written, you can use Astro Docs Docs as a general example: https://contribute.docs.astro.build/.
```

Comment:
is there a better way we could do the link?  it is mdx... I am not a fan have the `.` at the end of the URL

Reply:
ah yeah didn't notice it! nwtwwhb but we'll fix that tomorrow!